packages/mendel-browser-pack/index-deps.js

```diff
@@ -0,0 +1,106 @@
+/**
+ * Modules internal to a bundle can be referenced by index instead of id.
+ * Remap ids and deps in such case.
+ * @example
+ * This compresses the bundle by renaming all dependency indexes from file paths
+ * to a numbered index.
+ *
+ * Here is a sample transformation:
+ * ```js
```

Comment:
I know, but this is also from when Mendel didn't even had a notion of generators.